### ISSUE TITLE
It's ok to get all this warnings?

### ISSUE THREAD

**Issue by ZeroCool22** [AliaksandrSiarohin/first-order-model#241]
(FOM) C:\Users\ZeroCool22\Documents\FOM\first-order-model>python demo.py  --config config/vox-adv-256.yaml --driving_video driving_video/crop.mp4 --source_image source_image/source.png --checkpoint fom_checkpoints/vox-adv-cpk.pth.tar --relative --adapt_scale
C:\Users\ZeroCool22\anaconda3\envs\FOM\lib\site-packages\skimage\transform\_warps.py:105: UserWarning: The default mode, 'constant', will be changed to 'reflect' in skimage 0.15.
  warn("The default mode, 'constant', will be changed to 'reflect' in "
C:\Users\ZeroCool22\anaconda3\envs\FOM\lib\site-packages\skimage\transform\_warps.py:110: UserWarning: Anti-aliasing will be enabled by default in skimage 0.15 to avoid aliasing artifacts when down-sampling images.
  warn("Anti-aliasing will be enabled by default in skimage 0.15 to "
demo.py:27: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  config = yaml.load(f)
C:\Users\ZeroCool22\anaconda3\envs\FOM\lib\site-packages\torch\nn\functional.py:3000: UserWarning: The default behavior for interpolate/upsample with float scale_factor changed in 1.6.0 to align with other frameworks/libraries, and uses scale_factor directly, instead of relying on the computed output size. If you wish to keep the old behavior, please set recompute_scale_factor=True. See the documentation of nn.Upsample for details.
  warnings.warn("The default behavior for interpolate/upsample with float scale_factor changed "
  0%|                                                                                           | 0/76 [00:00<?, ?it/s]C:\Users\ZeroCool22\anaconda3\envs\FOM\lib\site-packages\torch\nn\functional.py:3384: UserWarning: Default grid_sample and affine_grid behavior has changed to align_corners=False since 1.3.0. Please specify align_corners=True if the old behavior is desired. See the documentation of grid_sample for details.
  warnings.warn("Default grid_sample and affine_grid behavior has changed "
C:\Users\ZeroCool22\anaconda3\envs\FOM\lib\site-packages\torch\nn\functional.py:3121: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  "See the documentation of nn.Upsample for details.".format(mode))
C:\Users\ZeroCool22\anaconda3\envs\FOM\lib\site-packages\torch\nn\functional.py:1625: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
100%|██████████████████████████████████████████████████████████████████████████████████| 76/76 [00:02<00:00, 30.89it/s]
C:\Users\ZeroCool22\anaconda3\envs\FOM\lib\site-packages\skimage\util\dtype.py:130: UserWarning: Possible precision loss when converting from float32 to uint8
  .format(dtypeobj_in, dtypeobj_out))
 
(FOM) C:\Users\ZeroCool22\Documents\FOM\first-order-model>

**Comment by AliaksandrSiarohin** [comment]
Yes.

**Comment by ZeroCool22** [comment]
> 
> 
> Yes.

Thanks!
